From 50b2c1570573823e3d8f7d6b0bbb3259010b2710 Mon Sep 17 00:00:00 2001
From: Debasish Biswas <debasishbsws.dev@gmail.com>
Date: Fri, 17 Jan 2025 18:59:39 +0530
Subject: [PATCH 2/5] Fix(build): Removing the version constraint from kfp to resolve
 build issue

ERROR: No matching distribution found for kfp==2.0.1, this is persisted from the start of the package
Signed-off-by: Debasish Biswas <debasishbsws.dev@gmail.com>
---
 backend/requirements.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/backend/requirements.txt b/backend/requirements.txt
index 889feb4c4..ba27bcb21 100644
--- a/backend/requirements.txt
+++ b/backend/requirements.txt
@@ -43,7 +43,7 @@ idna==3.7
     # via requests
 importlib-metadata==6.7.0
     # via click
-kfp==2.0.1
+kfp
     # via -r -
 kfp-pipeline-spec==0.2.2
     # via kfp
-- 
2.39.5 (Apple Git-154)

